# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/GfxInfo.h
# Commit: f559b920572b
# Full Hash: f559b920572b19d1f943744416c38baf82780b98
# Author: Bert Peers <bpeers@mozilla.com>
# Date: 2020-07-04 09:16:58
# Regressor Bug: 1649490
# File Overlap Count: 2
# Description:
#   Bug 1649490 - detect all Mac GPUs r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D82225
# ==============================================================================

diff -r ffb48be9b002 -r f559b920572b widget/cocoa/GfxInfo.h
--- a/widget/cocoa/GfxInfo.h	Sat Jul 04 00:16:53 2020 +0300
+++ b/widget/cocoa/GfxInfo.h	Fri Jul 03 21:11:02 2020 +0000
@@ -81,14 +81,16 @@
   void GetSelectedCityInfo();
   void AddCrashReportAnnotations();
 
-  uint32_t mAdapterRAM;
-  nsString mDeviceID;
-  nsString mDriverVersion;
-  nsString mDriverDate;
-  nsString mDeviceKey;
+  uint32_t mNumGPUsDetected;
 
-  nsString mAdapterVendorID;
-  nsString mAdapterDeviceID;
+  uint32_t mAdapterRAM[2];
+  nsString mDeviceID[2];
+  nsString mDriverVersion[2];
+  nsString mDriverDate[2];
+  nsString mDeviceKey[2];
+
+  nsString mAdapterVendorID[2];
+  nsString mAdapterDeviceID[2];
 
   uint32_t mOSXVersion;
 };